src/Rules/UnusedFunctionParametersCheck.php

```diff
@@ -43,20 +41,18 @@ public function getUnusedParameters(
 		string $identifier,
 	): array
 	{
-		$parameterNames = array_map(static function (Variable $variable): string {
+		$unusedParameters = [];
+		foreach ($parameterVars as $variable) {
 			if (!is_string($variable->name)) {
 				throw new ShouldNotHappenException();
 			}
-			return $variable->name;
-		}, $parameterVars);
-		$unusedParameters = array_combine($parameterNames, $parameterVars);
```

Comment:
rewrite/simplifed this code, as it relied on `array_combine` which can return `false` on PHP7.4, which got caught by the MutatingScope improvement.

> Argument of an invalid type array<string, PhpParser\Node\Expr\Variable>|false supplied for foreach, only iterables are supported.

rewrote it to get rid of the PHP 7.4 specific behaviour so I don't need to add 7.4 only ignores or similar hacks.